mopidy_alsamixer/mixer.py

```diff
@@ -70,13 +74,20 @@ def get_volume(self):
         if not channels:
             return None
         elif channels.count(channels[0]) == len(channels):
-            return int(channels[0])
+            unadjusted_volume = channels[0]
+            if self.logarithmic_volume:
+                unadjusted_volume = 100.0 * math.pow(unadjusted_volume / 100.0, 3.0)
```

Comment:
![screen shot 2017-02-03 at 9 06 15 am](https://cloud.githubusercontent.com/assets/1086005/22566308/3498a94e-e9f0-11e6-86f9-0d959f706756.png)

Graph comparing the logarithmic and cubic volume scales. Cubic is in black. Logarithmic in grey.